packages/next-tinacms-cloudinary/CHANGELOG.md

```diff
@@ -0,0 +1,265 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
```

Comment:
Shouldn't this file only start out with the intro lines up to here, and nothing else?